pkg/minikube/cruntime/containerd.go

```diff
@@ -33,6 +34,19 @@ func (r *Containerd) Name() string {
 	return "containerd"
 }
 
+// Version retrieves the current version of this runtime
+func (r *Containerd) Version() string {
+	ver, err := r.Runner.CombinedOutput("containerd --version")
+	// containerd github.com/containerd/containerd v1.2.0 c4446665cb9c30056f4998ed953e6d4ff22c7c39
+	if err == nil {
```

Comment:
Try to keep the normal code path at a minimal indentation by terminating the exceptional condition early on. For instance:

if err != nil {
  return err
}
<everything else>

Reply:
Done, just returning empty string to keep the API simple